internal/services/aws_reservation_service.go

```diff
@@ -52,19 +53,16 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Validate architecture match (hardcoded since image builder currently only supports x86_64). This can be only done
-	// when launch template is not set.
-	if payload.LaunchTemplateID == "" {
```

Comment:
:warning: I did not undesrtand why, so I've removed the condition for LaunchTemplate being empty :thinking: